spec/spec_helper.rb

```diff
@@ -18,6 +18,8 @@
   add_group "Database", "/db"
 end
 
+system('cat /etc/os-release')
```

Comment:
Ubuntu "latest" whatever that github thinks that is today 😅 Set here - https://github.com/rubyforgood/casa/blob/c74097fab2ea5461c421f02b31fb6f5578b39978/.github/workflows/rspec.yml#L22